### DISCUSSION TITLE
[Feature request][Balance] Nerfing a Humpback

### DISCUSSION THREAD

**Discussion by AbyssBeast09** [FakeFishGames/Barotrauma#9442] [Ideas and suggestions]
What about nerfing Humpback a little bit? For me its very op for starter sub(Having all fabricators, deconstructor, 360 gun cover) 

Im thinking about removing all lower guns and changing railgun to hardpoint 
You can put on top a Railgun and one Coilgun/chaingun 

And what about removing the medfab? 
![Screenshot_59](https://user-images.githubusercontent.com/85645359/174899018-e77e7c19-10fc-493c-8cf8-e8eb2e1d496c.png)

**Comment by Ek-Videogames** [comment]
I mean you could just instead make the humpback part of the 15K to 20k price tier, and swap it with one of the scuffed submarines like the Typhon2 or something.

Humpback is not "Overpowered" its "Campaign-Viable", if there were less vanilla submarines that didn't cost more it while doing less
( Crafting, storage locker capacity, weapons/hardpoints ) then there would be valid for more people to use subs besides the humpback.
Most of the time shelling out the 20k marks to get one of the "better" submarines feels like a direct downgrade because of this.
